### ISSUE TITLE
Finetune ALBERT using pretained model

### ISSUE THREAD

**Issue by astrongstorm** [google-research/google-research#121]
Hi,

Thanks very much for releasing the pre-trained code for ALBERT. I have downloaded THhub model from https://tfhub.dev/google/albert_base/1 , However, I found the accuracy of CoLA and MNLI seems to indicate, this is model does not contain pre-trained weight fro the ALBERT. 
I have tried:

python3 -m run_classifier_sp --data_dir=../../DataSet/CoLA/ --task_name=cola —-output_dir=testing_sp_modifying_model_setting_in_main_file --vocab_file=vocab.txt --albert_config_file=2/assets/albert_config.json --do_train=False --do_eval=True --max_seq_length=128  --train_batch_size=32  --learning_rate=2e-05 --num_train_epochs=3.0

with do_train set to False, getting the accuracy of around 40%,
however, with do_train set to True, I am able to get the accuracy of 70%

Besides, after downloading the weight from TH_Hub, I can only find the .json file, but without the .ckpt file from the folder.

I am wondering, how can we find the per-trained weight, the .ckpt file for ALBERT?

**Comment by AshutoshUpadhya** [comment]
I am facinig the same issue. There is no .ckpt file . Is there a way to get to get the model weights file (model.data,index,meta) from the TFHub model tar file.

Thanks in advance

**Comment by chiragsanghvi10** [comment]
I am facing the same issue. any idea on this?

**Comment by dhruvsakalley** [comment]
![image](https://user-images.githubusercontent.com/3661965/69497281-30403e80-0ea9-11ea-80a7-d4cceeff1ba0.png)

**Comment by VikasRajashekar** [comment]
where did you find the vocab.txt file?

**Comment by ZheyuYe** [comment]
> where did you find the vocab.txt file?

the vocab.txt file for version 1 can be found in https://tfhub.dev/google/albert_base/2 where the only difference bewteen this two versions is the mechanism of dropout leaving the spm and vocab as same.
